client/src/app/core/actions/user-action.ts

```diff
@@ -1,8 +1,70 @@
+import { HasMeetingId } from 'app/shared/models/base/has-meeting-id';
 import { Identifiable } from 'app/shared/models/base/identifiable';
+import { Decimal, Id, UnsafeHtml } from '../definitions/key-types';
 
 export namespace UserAction {
-    export interface CreatePayload {}
-    export interface UpdatePayload {}
-    export interface ResetPasswordPayload extends Identifiable {}
-    export interface SetNewPasswordPayload extends Identifiable {}
+    interface PartialTemporaryUserPayload {
+        // Optional:
+        title?: string;
+        first_name?: string;
+        last_name?: string;
+        is_active?: boolean;
+        is_committee?: boolean;
+        default_password?: string;
+        about_me?: UnsafeHtml;
+        gender?: string;
+        comment?: UnsafeHtml;
+        number?: string;
+        structure_level?: string;
+        email?: string;
+        vote_weight?: Decimal;
+
+        is_present_in_meeting_ids?: Id[]; // can only contain the given meeting_id
+        group_meeting_ids?: Id[] | string[]; // the structured field is only valid for the meeting_id
```

Comment:
for partial users we decided just to use `group_ids?: Id[]`. The backend will put it into the right template field